sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -88,6 +89,10 @@ class DynamoDBOnlineStore(OnlineStore):
     _dynamodb_resource = None
     _aioboto_session = None
 
+    @property
+    def async_supported(self) -> SupportedAsyncMethods:
+        return SupportedAsyncMethods(read=True)
```

Comment:
the default impl of this property has

```
online:
    read: False
    write: False
offline:
    read: False
    write: False
```

override and mark supported operations as they're added